### PR TITLE
docs: add nextjs links

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

add nextjs links to intro articles.

## Testing

Local preview

### CODE REVIEW

File: docs/docs/tutorials/get-started/create-and-integrate-the-first-app.mdx [hunk 1]
```diff
@@ -55,7 +55,10 @@ If you skip the guide in Admin Console by accident, you can also pick the guide
     },
     {
       title: 'Tranditional Web App',
-      items: [{ key: 'Traditional Web', link: '/docs/recipes/integrate-logto/traditional' }],
+      items: [
+        { key: 'Next.js', link: '/docs/recipes/integrate-logto/next-js' },
+        { key: 'Traditional Web', link: '/docs/recipes/integrate-logto/traditional' },
```
Comment (IceHe) [inline docs/docs/tutorials/get-started/create-and-integrate-the-first-app.mdx:60]: The links should end with `.md` or `.mdx`?

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/tutorials/get-started/create-and-integrate-the-first-app.mdx [hunk 2]
```diff
@@ -55,7 +55,10 @@ import ApplicationTypesIntro from '../../fragments/_application_types_intro.mdx'
     },
     {
       title: '传统 web 应用',
-      items: [{ key: '传统 web', link: '/zh-cn/docs/recipes/integrate-logto/traditional' }],
+      items: [
+        { key: 'Next.js', link: '/zh-cn/docs/recipes/integrate-logto/next-js' },
+        { key: '传统 web', link: '/zh-cn/docs/recipes/integrate-logto/traditional' },
```
Comment (IceHe) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/tutorials/get-started/create-and-integrate-the-first-app.mdx:60]: ditto

**Review by IceHe — APPROVED** [review]
Overall LGTM

File: docs/docs/tutorials/get-started/create-and-integrate-the-first-app.mdx (hunk 1 above)
Comment (demonzoo) [inline docs/docs/tutorials/get-started/create-and-integrate-the-first-app.mdx:60]: I think we should change `Traditional Web` to either `Express` or `Node.js`, or `General` in the long term

Comment (wangsijie) [reply docs/docs/tutorials/get-started/create-and-integrate-the-first-app.mdx:60]: Yes, we can take some time to think about it.

Comment (wangsijie) [reply docs/docs/tutorials/get-started/create-and-integrate-the-first-app.mdx:60]: Just align with current code.

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/tutorials/get-started/create-and-integrate-the-first-app.mdx (hunk 2 above)
Comment (wangsijie) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/tutorials/get-started/create-and-integrate-the-first-app.mdx:60]: Just align with current code.

**Pull request merged**